### PR TITLE
ts/utilites-namespace-functions

### PR DESCRIPTION
**Pull request by marvin19**
Removed functions from internal namespace in Utilities.

### CODE REVIEW

**Review by bre1470 — APPROVED** [review]
Thank you! Looks good. 👍

File: ts/Core/Animation/Fx.ts [hunk 1]
```diff
@@ -533,6 +541,19 @@ interface Fx extends FxLike {
...
 }
 
+/* *
+ *
+ *  Compatibility
+ *
+ * */
+
 H.Fx = Fx;
+(H as any).timers = Fx.timers;
```
Comment (TorsteinHonsi) [inline ts/Core/Animation/Fx.ts:551]: I don't think we need to keep this on the namespace. If the tests pass without it I'd say we remove it.

**Review by TorsteinHonsi — CHANGES_REQUESTED** [review]

File: ts/Core/Animation/Fx.ts (hunk 1 above)
Comment (bre1470) [reply ts/Core/Animation/Fx.ts:551]: test-utilities requires an update then. (highcharts and highcharts-utils repositories)

Comment (TorsteinHonsi) [reply ts/Core/Animation/Fx.ts:551]: Okay. We'll leave it for now then.

**Review by TorsteinHonsi — APPROVED** [review]

**Pull request merged**